client/src/components/shared/header.tsx

```diff
@@ -0,0 +1,37 @@
+import AppBar from "@mui/material/AppBar";
+import Box from "@mui/material/Box";
+import Toolbar from "@mui/material/Toolbar";
+import Typography from "@mui/material/Typography";
+import Button from "@mui/material/Button";
+
+const Header = () => {
+  return (
+    <Box flexGrow={1}>
+      <AppBar position="static" >
+        <Toolbar sx={{ marginX: "50px" }}>
```

Comment:
I think the margin is not needed. But if we need it we should extend and define values in our theme, so that we don't have random numbers in each component.